src/views/Stake/Stake.jsx

```diff
@@ -259,10 +272,9 @@ function Stake() {
                       //hides the tab underline sliding animation in while <Zoom> is loading
                       TabIndicatorProps={!zoomed && { style: { display: "none" } }}
                     >
-                      <Tab label="Stake" {...a11yProps(0)} />
-                      <Tab label="Unstake" {...a11yProps(1)} />
+                      <Tab label={t`do_stake`} {...a11yProps(0)} />
```

Comment:
should this be
```
t`Stake`
```

Reply:
I had to differentiate "Stake" the noun and "Stake" the verb because in many language they are not translated the same. Same for bonds.